### ISSUE TITLE
Error when spaces in path of cwd

### ISSUE THREAD

**Issue by JoshKloster** [mtscout6/syntastic-local-eslint.vim#6]
If there are spaces in the cwd I get the following error

```
Error detected while processing /home/josh/.vim/plugged/syntastic-local-eslint.vim/ftplugin/javascript.vim:          
line    4:                                                                                                           
E172: Only one file name allowed: lcd /home/josh/Documents/test dir/video-service 
```

But if "test dir" is changed to "test-dir" everything works perfectly.

**Comment by mtscout6** [comment]
If you feel up to it feel free to submit a PR with a bug fix and I'll get that merged in.

**Comment by JoshKloster** [comment]
On it.

**Comment by mtscout6** [comment]
Resolved by #7
